_inc/lib/debugger/class-jetpack-cxn-test-base.php

```diff
@@ -49,16 +49,21 @@ public function __construct() {
 	/**
 	 * Adds a new test to the Jetpack Connection Testing suite.
 	 *
+	 * @since 7.1
+	 * @since 7.2 Added name arg.
+	 *
 	 * @param callable $callable Test to add to queue.
 	 * @param array    $groups Testing groups to add test to.
+	 * @param string   $name Common name for the test.
 	 *
 	 * @return bool True if successfully added. False for a failure.
 	 */
-	public function add_test( $callable, $groups = array( 'default' ) ) {
+	public function add_test( $callable, $groups = array( 'default' ), $name = '' ) {
 		if ( is_callable( $callable ) ) {
 			$this->tests[] = array(
 				'test'  => $callable,
 				'group' => $groups,
+				'name'  => $name,
```

Comment:
I intentionally didn't set it as the key of the array to prevent name collision.